src/app/index.scss

```diff
@@ -9,17 +9,104 @@ $fa-font-path: "../../bower_components/fontawesome/fonts";
 @import "../../bower_components/fontawesome/scss/font-awesome.scss";
 @import "../../bower_components/angular-tooltips/dist/angular-tooltips";
 
+
 .top{
   z-index: 100;
 }
 
+/* prevent element from being selected http://stackoverflow.com/questions/826782 */
+.noselect {
+  user-select: none;
+  cursor: default;
+}
+
+.voyager {
+  background-color: #ECF1F7;
+}
+
+.voyager a {
+  color: #38425D;
+}
+
+.voyager a:hover {
+  color: #72757B;
+}
+
+h2 {
+  @extend .noselect;
+  font-size: 16px;
+  line-height: 20px;
+  font-weight: 700;
+  margin: 0 0 15px;
+}
+
+#logo {
+  height: 34px;
+  flex-shrink: 0;
+}
+
+.controls {
+  margin-left: 33px;
+  margin-top: 5px;
+  a {
+    margin: 10px;
```

Comment:
I guess we should remove `margin: 10px` as it doesn’t have any effect on the screen from my test.   
(`margin-top|bottom` do not matter here and `.command` already defines `margin-left`, while `margin-right` is overridden on line 54).

Please let me know if I overlook anything.